src/ludamus/templates/notice_board/index.html

```
@@ -61,7 +61,7 @@ <h2 class="text-xl font-bold mb-4 text-foreground">{% translate "Past" %}</h2>
     </section>
 {% endblock body %}
 {% block extra_scripts %}
-    <script>
+    <script {% if csp_nonce is not None %} nonce="{{ csp_nonce }}"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)